car/models.py

```diff
@@ -0,0 +1,54 @@
+from django.db import models
+
+class CarModel(models.Model): 
+    name = models.CharField(max_length = 64)
+
+    class Meta: 
+        db_table = 'car_models'
+
+class Drive(models.Model): 
+    name = models.CharField(max_length = 64)
+
+    class Meta: 
+        db_table = 'drives'
+
+class Battery(models.Model): 
+    name = models.CharField(max_length = 64)
+
+    class Meta: 
+        db_table = 'batteries'
+
+class CarSpecification(models.Model): 
+    model            = models.ForeignKey(CarModel, on_delete = models.CASCADE)
+    drive            = models.ForeignKey(Drive, on_delete = models.CASCADE)
+    battery          = modles.ForeignKey(Battery, on_delete = models.CASCADE)
+    traction_ability = models.IntegerField(null = True)
+    height           = models.IntegerField(null = True)
+    vault_length     = models.IntegerField(null = True)
+    approach_angle   = modles.IntegerField(null = True)
+    deviation_angle  = models.IntegerField(null = True)
+    distance         = models.IntegerField(null = True)
+    load_space       = models.IntegerField(null = True)
+    acceleration     = models.DecimalField(max_digits = 8, decimal_places = 2)
+    top_speed        = models.DecimalField(max_digits = 8, decimal_places = 2)
+    weight           = models.IntegerField(null = True)
+    display_inch     = models.IntegerField(null = True)
+    seats            = models.IntegerField(null = True)
+    warranty         = models.CharField(max_length = 256)
+
+    class Meta:
+        db_table = 'car_specifications'
+
+
+class WheelInch(models.Model):
+    inch          = models.IntegerField()
+    
+    class Meta:
+        db_table = 'wheel_inches'
+
+class CarSpecWheel(models.Model):
+    wheel     = models.ForeignKey(WheelInch, on_delete = models.CASCADE)
```

Comment:
위에서 변경 후 아래에도 맞게 수정해주세요~~
```suggestion
    wheel     = models.ForeignKey(Wheel, on_delete = models.CASCADE)
```

Reply:
네!!!